index.md

```diff
@@ -6,6 +6,6 @@ Collaborative software engineer
 
 *Collabortive Software Project in Computational Physics
 
-##My skills
+###My skills
```

Comment:
```suggestion
This is a better change 
```
This would be a better change to the change that you did because it is changing a lot more